dg/templates/output/jslps_adoption_module.html

```diff
@@ -0,0 +1,344 @@
+{% extends "base_jslps_analytics.html" %} {% load analytics_extra_tags %} {% block title %} Digital Green - Adoption Statistics {% endblock %} {% block cssfile %} {{block.super}}
```

Comment:
we can send base_template_name from views and use that in extends method. By doing this, we wont have to re-write the same file again.

Reply:
I have discussed with you why we are not following that approach for now.